src/components/tl/mlx5/mcast/tl_mlx5_mcast_helper.c

```diff
@@ -571,23 +553,28 @@ ucc_status_t ucc_tl_mlx5_mcast_modify_rc_qps(ucc_tl_mlx5_mcast_coll_context_t *c
     return UCC_OK;
 }
 
-ucc_status_t ucc_tl_mlx5_fini_mcast_group(ucc_tl_mlx5_mcast_coll_context_t *ctx,
-                                          ucc_tl_mlx5_mcast_coll_comm_t    *comm)
+ucc_status_t ucc_tl_mlx5_leave_mcast_group(ucc_tl_mlx5_mcast_coll_context_t *ctx,
+                                           ucc_tl_mlx5_mcast_coll_comm_t    *comm)
 {
     char        buf[40];
```

Comment:
should be INET6_ADDRSTRLEN at least, otherwise it might be overflow for some addresses